### PR TITLE
[meteoblue] update readme.md for beginners

### PR DESCRIPTION
**Pull request by mcmichels**
As my code-duplication in #6302 was too long and i understood the problem of duplication of lines i want to bring in just a few lines for demonstration.

I hope that beginners have it easier to understand how to duplicate code for all days now :-)

This time correctly signed hopefully.

### CODE REVIEW

**Comment by 9037568** [comment]
No, for all the same reasons.

**Comment by kaikreuzer** [comment]
@9037568 Let us please discuss this a bit more detailed with @mcmichels - noting that he wants to solve a user issue with this.

Most of this PR is just adding identation to the existing example - are you against this as well?
What's wrong about the small short explanatory sentence that he has added at the end?

File: bundles/org.openhab.binding.meteoblue/README.md [hunk 1]
```diff
@@ -112,37 +112,44 @@ demo.items:
...
+Number:Pressure		day0PressH			"High Pressure [%d %unit%]"			<pressure>		(weatherDay0)		{channel="meteoblue:weather:metBridge:A51:forecastToday#maxSeaLevelPressure"}
+Number			day0WindDir			"Wind Direction [%d]"				<wind>			(weatherDay0)		{channel="meteoblue:weather:metBridge:A51:forecastToday#windDirection"}
+String			day0CWindDir			"Cardinal Wind Direction [%s]"			<wind>			(weatherDay0)		{channel="meteoblue:weather:metBridge:A51:forecastToday#cardinalWindDirection"}
+Number:Speed		day0WindSpL			"Low Wind Speed [%.2f mph]"			<wind>			(weatherDay0)		{channel="meteoblue:weather:metBridge:A51:forecastToday#minWindSpeed"}
+Number:Speed		day0WindSpH			"High Wind Speed [%.2f mph]"			<wind>			(weatherDay0)		{channel="meteoblue:weather:metBridge:A51:forecastToday#maxWindSpeed"}
+
 ```
+To setup the values for the next days follow the schema from above. For day 4 one example:
+<pre>
+<code>// ----------------- Day4 ------------------------------------------------------
```
Comment (Hilbrand) [inline bundles/org.openhab.binding.meteoblue/README.md]: Use ``` for code examples.

File: bundles/org.openhab.binding.meteoblue/README.md [hunk 2]
```diff
@@ -112,37 +112,44 @@ demo.items:
...
+Number:Length		day0Snow			"Snowfall [%.2f in]"				<rain>			(weatherDay0)		{channel="meteoblue:weather:metBridge:A51:forecastToday#snowFall"}
+Number:Pressure		day0PressL			"Low Pressure [%d %unit%]"			<pressure>		(weatherDay0)		{channel="meteoblue:weather:metBridge:A51:forecastToday#minSeaLevelPressure"}
+Number:Pressure		day0PressH			"High Pressure [%d %unit%]"			<pressure>		(weatherDay0)		{channel="meteoblue:weather:metBridge:A51:forecastToday#maxSeaLevelPressure"}
+Number			day0WindDir			"Wind Direction [%d]"				<wind>			(weatherDay0)		{channel="meteoblue:weather:metBridge:A51:forecastToday#windDirection"}
+String			day0CWindDir			"Cardinal Wind Direction [%s]"			<wind>			(weatherDay0)		{channel="meteoblue:weather:metBridge:A51:forecastToday#cardinalWindDirection"}
+Number:Speed		day0WindSpL			"Low Wind Speed [%.2f mph]"			<wind>			(weatherDay0)		{channel="meteoblue:weather:metBridge:A51:forecastToday#minWindSpeed"}
+Number:Speed		day0WindSpH			"High Wind Speed [%.2f mph]"			<wind>			(weatherDay0)		{channel="meteoblue:weather:metBridge:A51:forecastToday#maxWindSpeed"}
+
 ```
+To setup the values for the next days follow the schema from above. For day 4 one example:
```
Comment (Hilbrand) [inline bundles/org.openhab.binding.meteoblue/README.md]: Please put each sentence on a single line.

**Comment by 9037568** [comment]
````
To setup the values for the next days follow the schema from above. 
````

This is stating the blindingly obvious. 

Then below this we copy one of the preceding item definitions and change two characters in it. 
I don't get how this adds any value.

Those are my 2 drachmas (adjusted for inflation).

File: bundles/org.openhab.binding.meteoblue/README.md (hunk 1 above)
Comment (mcmichels) [reply bundles/org.openhab.binding.meteoblue/README.md]: but within ``` i cannot have bold text when i remember correctly. i used bold text to show what has to be changed.

**Comment by mcmichels** [comment]
> ```
> To setup the values for the next days follow the schema from above. 
> ```
> 
> This is stating the blindingly obvious.
> 
> Then below this we copy one of the preceding item definitions and change two characters in it.
> I don't get how this adds any value.
> 
> Those are my 2 drachmas (adjusted for inflation).

If this is your opinion i will close the pull request. But with this opinion you block total beginners to use your documentation. OpenHab is not easy when you are totally new to it.

**Pull request closed without merging**